ch01.asciidoc

```diff
@@ -6,18 +6,18 @@
 
 The Lightning Network is a payment system for Bitcoin.
```

Comment:
```suggestion
The Lightning Network (LN) is a payment system for Bitcoin.
```

Reply:
The abbreviation should be introduced at the first usage (dismiss my commit suggestion if it was already introduced previously and I missed it.)